packages/moonstone/TimePicker/TimePickerBase.js

```diff
@@ -329,6 +363,7 @@ const TimePickerBase = kind({
 								case 'a':
 									return (
 										<DateComponentPicker
+											aria-label={ariaLabel}
```

Comment:
Is there a reason we wouldn't support individual props (`meridiemAriaLabel` and `meridiemAccessibilityHint`) for meridiem too?

Reply:
I did't add TimePicker meridiem labels because there was no scenario to read.
I'll add meridiemAriaLabel and meridiemAccessibilityHint.